From fe422b5933b23ff8a035cf2d42200e8ef5903a9c Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Thu, 14 Feb 2019 16:01:26 +0100
Subject: FileSystemURL. do not declare noexcept in move constructor

GCC fails to build as it cannot resolve as noexcept all its dependencies.

Change-Id: I0f97896087244929fe63a323d1e9e53e359eaf14

diff --git a/storage/browser/fileapi/file_system_url.cc b/storage/browser/fileapi/file_system_url.cc
index b90cf29045c7..e28b1f85d8f6 100644
--- a/storage/browser/fileapi/file_system_url.cc
+++ b/storage/browser/fileapi/file_system_url.cc
@@ -26,7 +26,7 @@ FileSystemURL::FileSystemURL()
 
 FileSystemURL::FileSystemURL(const FileSystemURL& other) = default;
 
-FileSystemURL::FileSystemURL(FileSystemURL&& other) noexcept = default;
+FileSystemURL::FileSystemURL(FileSystemURL&& other) = default;
 
 FileSystemURL& FileSystemURL::operator=(FileSystemURL&& rhs) = default;
 
diff --git a/storage/browser/fileapi/file_system_url.h b/storage/browser/fileapi/file_system_url.h
index 8220812d0333..f6b8813e2d73 100644
--- a/storage/browser/fileapi/file_system_url.h
+++ b/storage/browser/fileapi/file_system_url.h
@@ -82,7 +82,7 @@ class COMPONENT_EXPORT(STORAGE_BROWSER) FileSystemURL {
   FileSystemURL(const FileSystemURL& other);
   // Constructs FileSystemURL with the contents of |other|, which is left in
   // valid but unspecified state.
-  FileSystemURL(FileSystemURL&& other) noexcept;
+  FileSystemURL(FileSystemURL&& other);
   ~FileSystemURL();
 
   // Replaces the contents with those of |rhs|, which is left in valid but
-- 
2.17.1

